packages/studio-base/src/components/ExtensionsSettings/index.tsx

```diff
@@ -152,22 +160,21 @@ export default function ExtensionsSettings(): React.ReactElement {
     );
   }
 
-  if (marketplaceEntries.error) {
-    return (
-      <Stack gap={1} alignItems="center" justifyContent="center" fullHeight>
-        <Typography align="center" variant="body2" color="text.secondary">
-          Failed to fetch the list of available extensions. Check your Internet connection and try
-          again.
-        </Typography>
-        <Button onClick={async () => await refreshMarketplaceEntries()}>
-          Retry Fetching Extensions
-        </Button>
-      </Stack>
-    );
-  }
-
   return (
     <Stack gap={1}>
+      {marketplaceEntries.error && (
+        <Alert severity="error">
+          <Stack gap={1} alignItems="center" justifyContent="center" fullHeight>
+            <Typography align="center" variant="body2" color="text.secondary">
+              Failed to fetch the list of available marketplace extensions. Check your Internet
+              connection and try again.
+            </Typography>
```

Comment:
Use the AlertTitle component
```suggestion
            <AlertTitle>Failed to fetch the list of available marketplace extensions,</AlertTitle>
            Check your Internet connection and try again.
```